pkg/graph/schema.resolvers.go

```diff
@@ -883,6 +883,64 @@ func (r *mutationResolver) UpdateSystemIntakeReviewDates(ctx context.Context, in
 	}, err
 }
 
+func (r *mutationResolver) UpdateSystemIntakeContactDetails(ctx context.Context, input model.UpdateSystenIntakeContactDetailsInput) (*model.UpdateSystemIntakePayload, error) {
+	intake, err := r.store.FetchSystemIntakeByID(ctx, input.ID)
+	if err != nil {
+		return nil, err
+	}
+	intake.Requester = input.Requester.Name
+	intake.Component = null.StringFrom(input.Requester.Component)
+	intake.BusinessOwner = null.StringFrom(input.BusinessOwner.Name)
+	intake.BusinessOwnerComponent = null.StringFrom(input.BusinessOwner.Component)
+	intake.ProductManager = null.StringFrom(input.ProductManager.Name)
+	intake.ProductManagerComponent = null.StringFrom(input.ProductManager.Component)
+
+	if input.Isso.IsPresent {
+		intake.ISSOName = null.StringFrom(*input.Isso.Name)
+	}
+
+	if !input.Isso.IsPresent {
+		intake.ISSOName = null.StringFromPtr(nil)
+	}
+
+	if input.GovernanceTeams.IsPresent {
+		trbCollaboratorName := null.StringFromPtr(nil)
+		for _, team := range input.GovernanceTeams.Teams {
```

Comment:
Is there an easier way to find the matching value and either set it to that or set it to nil?